### ISSUE TITLE
LDAP integration

### ISSUE THREAD

**Issue by wdavidw** [Mailu/Mailu#302]
I'm considering using Mailu and I'm wondering if you have any documentation related to the integration of its software stack with an LDAP directory ?

**Comment by kaiyou** [comment]
There is currently no LDAP support in Mailu, as components are still too tightly linked to the SQLite database. A step was taken by migrating to the admin container for all authentication tasks, but we still need to remove all direct references to the database from Postfix, Dovecot and Fetchmail before we can actually support other backends.

This is related to #83 btw :)

**Comment by wdavidw** [comment]
Sorry for the duplicate, i didn't realise that github search was only covering source code and not the issues. Closing in favor of #83.
